### ISSUE TITLE
Databases can be deleted without password

### ISSUE THREAD

**Issue by qweke** [MiniKeePass/MiniKeePass#357]
The iPhone app allows to delete a entire database (with a lot of passwords stored inside and protected by password), in a simple way, without asking a main password or whatever.

**Comment by sakeven** [comment]
It's too easy to delete a database. Very dangerous...
Hope to add a feature that require a password to delete db.

**Comment by jamisonbennett** [comment]
I agree, that it is too easy to delete a database. But I don't think a password should be required to delete a database. There are other ways a database can be deleted without requiring a password such as just deleting the app or entering the wrong PIN too man times. But I think something should be done to prevent deleting a database from accidentally happening. How about adding a confirmation prompt? See my pull request https://github.com/MiniKeePass/MiniKeePass/pull/632.
